### DISCUSSION TITLE
Is there anyone who can help with BLTouch adjustment?

### DISCUSSION THREAD

**Discussion by alsahime** [mriscoc/Ender3V2S1#860] [Q&A]
Hello, I am using the "Ender3V2-SKRME3V3-BLTUBL-MPC-20230312.bin" firmware on my Ender 3 V2 printer with a BTT SKR MINI E3 V3 motherboard and a BLTouch. However, I am using the VORON design and my BLTouch is slightly to the left compared to the previous setup, so the center of my print bed is not being detected. Can anyone help me with how to adjust it as shown in the picture?

https://ibb.co/L9jxLsp
https://ibb.co/LZH1m2f

**Comment by TheWebMachine** [comment]
The probe offset can be adjusted in the Probe menu. Did you already adjust it there?

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
